modules/drivers/presto-jdbc/test/metabase/test/data/presto_jdbc.clj

```diff
@@ -37,23 +37,34 @@
                              :type/TimeWithTZ             "TIME WITH TIME ZONE"}]
   (defmethod sql.tx/field-base-type->sql-type [:presto-jdbc base-type] [_ _] db-type))
 
+(defn dbdef->connection-details [database-name]
+  (let [base-details
```

Comment:
Having some information on how to test this stuff and what to ensure is present would be helpful. I have no idea how to test kerberos stuff and it seems like i need to set about 15 env vars

Reply:
I don't have any good idea how to document this. I couldn't run the docker image on my machine, so I relied on the CI tests. In the end, the CI tests seem to be the best documentation (and the source of truth) about how to set the variables. Currently, this can be seen in `.github/scripts/run-presto-kerberos-integration-test.sh`.